### PR TITLE
{AzureRESTAPISpecs} Broken link in Getting started with OpenAPI specifications content

### LINKED ISSUE #25243: Broken link in Getting started with OpenAPI specifications content
Steps
1. Browse [here](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/Getting%20started%20with%20OpenAPI%20specifications.md)
2. Click "Azure REST API guidelines"

Results in 404.

I believe the proper link is [here](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md).

### PR DESCRIPTION
**Pull request by navba-MSFT**
Fixes Azure/azure-rest-api-specs#25243

Reference docs: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/Getting%20started%20with%20OpenAPI%20specifications.md

Azure REST API guidelines hyper link is incorrect.

Incorrect Link: https://github.com/microsoft/api-guidelines/blob/vNext/Azure/Guidelines.md

Correct Link: https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md

# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: documentation/Getting started with OpenAPI specifications.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Currently, we only support `OpenAPI Specification 2.0` or `Swagger V2.0`.
 
 * **[`Understanding the GitHub flow`](https://guides.github.com/introduction/flow/)**
 * **[`CONTRIBUTING.md`](https://github.com/Azure/azure-rest-api-specs/blob/main/CONTRIBUTING.md)**
-* **[`Azure REST API guidelines`](https://github.com/microsoft/api-guidelines/blob/vNext/Azure/Guidelines.md)**
+* **[`Azure REST API guidelines`](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md)**
```
Comment (mikekistler) [inline documentation/Getting started with OpenAPI specifications.md:14]: Thanks for spotting the broken link, but this is the correct one.
```suggestion
* **[`Azure REST API guidelines`](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md)**
```

**Review by mikekistler — CHANGES_REQUESTED** [review]
Please fix new link.

**Comment by navba-MSFT** [comment]
@mikekistler Your suggestions have been incorporated and committed. Could you please review and merge once you get a chance ?

**Review by mikekistler — APPROVED** [review]
Looks good! 👍 

Thank you for this fix.

**Pull request merged**